clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -604,6 +621,10 @@ class CIRGetGlobalOpLowering
   mlir::LogicalResult
   matchAndRewrite(mlir::cir::GetGlobalOp op, OpAdaptor adaptor,
                   mlir::ConversionPatternRewriter &rewriter) const override {
+    if (op.getName() == "printf") {
```

Comment:
What's this about?

Reply:
Just some debugging test I forgot to remove. I'll patch it.